web/src/stream_edit_subscribers.ts

```diff
@@ -201,10 +243,22 @@ function subscribe_new_users({pill_user_ids}) {
         });
     }
 
-    function invite_failure(xhr) {
+    function invite_failure(xhr: JQuery.jqXHR): void {
         let message = "Failed to subscribe user!";
-        if (xhr.responseJSON?.msg) {
-            message = xhr.responseJSON.msg;
+
+        const failure_response_schema = z
+            .object({
+                msg: z.string(),
+                code: z.string(),
+                result: z.string(),
+            })
+            .safeParse(xhr.responseJSON);
```

Comment:
`.safeParse` does not return a “schema”.

- #30527

Reply:
Yeah. Bad naming on my part. Thanks for the fix.